addons/mail/static/tests/owl/component/chatter_topbar_tests.js

```diff
@@ -50,19 +50,94 @@ QUnit.module('ChatterTopbar', {
     }
 });
 
-QUnit.test('base rendering', async function (assert) {
-    assert.expect(9);
+QUnit.test('base rendering (forever loading attachments)', async function (assert) {
```

Comment:
Prefer a name that tells what you want to test, instead of naming what the test does.

Here, for instance, you want to make assertions on rendering **attachment counter while loading attachments** of a thread. We don't really care that the test needs to keeps loading attachments forever to make it happen!